sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -60,6 +60,8 @@ static TestProxy()
 
         private TestProxy(string proxyPath, bool debugMode = false)
         {
+            debugMode |= !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("PROXY_MANUAL_START"));
```

Comment:
Could we use the same env name, like PROXY_DEBUG_MODE?

Reply:
And possibly update the README?

Reply:
Sure. I'm looking for the capability!